docs/configure/reports/resource-metrics.md

```diff
@@ -10,11 +10,11 @@ To use an existing Prometheus installation pass the following flag to the Helm c
 --set resourcemetrics.address="http://prometheus-server.<namespace>"
 ```
 
-Alternatively, you can click on **Prometheus** from the [Report Hub](https://insights.docs.fairwinds.com/installation/report-hub/), and then:
+Alternatively, you can click on **Prometheus** from the [Report Hub](https://insights.docs.fairwinds.com/run/agent/report-hub/), and then:
```

Comment:
If you make this a relative link then we have a script that will catch these mistakes. `/run/agent/report-hub/`